cmd/system-probe/api/client/client_windows.go

```diff
@@ -7,6 +7,15 @@
 
 package client
 
-const (
-	netType = "tcp"
+import (
+	"net/http"
+
+	process_net "github.com/DataDog/datadog-agent/pkg/process/net"
```

Comment:
I think this import might be what is causing the `trace-agent` import explosion.

Reply:
I suspect the non-windows versions had these imports beforehand, since only windows seems to have changed.

Reply:
The use of this package in `trace-agent` seems to be coming from `pkg/config/fetcher`. I don't think `trace-agent` needs knowledge of `system-probe` in this way. I can fix this in another PR.

Reply:
Fixed in https://github.com/DataDog/datadog-agent/pull/29617